tests/Fable.Tests/Tests.fs

```diff
@@ -0,0 +1,12 @@
+module Tests
+
+open Hedgehog
+
+[<EntryPoint>]
+let main args =
+    // For now just test if everything compiles
+    property {
+        let! x = Gen.int (Range.linearBounded())
+        return x = x
+    } |> Property.check
+    0
```

Comment:
That's just a smoke test, right? In the sense that it runs just to verify the basic functionality (that Fable compilation succeeds).

Reply:
Yes. I put it in there just to see if fable-compiler can compile this project referencing Hedgehog. I commented out one of the `#if !FABLE_COMPILER` and got an error when trying to compile the test project with fable-compiler so it should work as a test if fable-compiler can compile Hedgehog sources.